src/components/dialog/index.js

```diff
@@ -131,7 +133,10 @@ class Dialog extends BaseComponent {
               style={this.styles.gestureContainer}
             >
               <TouchableWithoutFeedback>
-                {this.props.children}
+                <SafeAreaView style={{flex: 1}}>
```

Comment:
why are u using here `SafeAreaView` and not our `View` with the `useSafeArea` prop?

Reply:
I just wanted to be more explicit. A View has no meaning here since it was added only for the purpose of adding the safe area.